Modules/ExtendedPlayerControl.cs

```diff
@@ -349,6 +349,7 @@ public static void SetKillCooldown(this PlayerControl player, float time = -1f,
         }
         player.ResetKillCooldown();
     }
+    public static Vector2 Pos(this PlayerControl pc) => new(pc.transform.position.x, pc.transform.position.y);
```

Comment:
There is already code that does the same thing "`GetCustomPosition`"
So either rename it to `Position` or remove this

Reply:
oh I didn't know that, ty. Removed the pos() method